src/styles/main.scss

```diff
@@ -1 +1,59 @@
-// put code here
+$circleSize: 80vmin;
+
+body {
+  margin: 0;
```

Comment:
It is not very convenient to look at the watch because its at the very top of the page, add some space please.